### PR TITLE
Use newest configuration, fix web support

### LINKED ISSUE #3: Doesn't work on web
If I refresh on http://localhost:19008/stack/user/Wojciech/22, it goes to http://localhost:19008/stack/home immediately

### PR DESCRIPTION
**Pull request by sAleksovski**
- Update to latest Expo version
- Uses latest configuration
- Fix refresh on Profile screen by setting `initialRouteName` in `HomeStack` (Should close #3 )

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,64 +1,45 @@
...
-        setIsReady(true);
-      });
-  }, [getInitialState]);
-
-  if (!isReady) {
-    return null;
-  }
 
+export default function App() {
   function Home({ navigation }) {
```
Comment (WoLewicki) [inline App.tsx:36]: I think the convention is to make the functions in the main scope of the file, not inside the `App` function. It is also better for reuse. Could you change it back?

Comment (sAleksovski) [reply App.tsx]: They were already inside `App`.
I tried to make a minimal change for easier review.

Anyway, I moved them now.

Comment (WoLewicki) [reply App.tsx]: Oh, I am sorry then. But I think they shouldn't be there so it is good now.

**Comment by WoLewicki** [comment]
Thanks for your contribution!

**Pull request merged**